_docs/install/kubernetes/dynamic-provisioning.md

```diff
@@ -9,11 +9,12 @@ module: install/kubernetes/dynamic-provisioning
 
 StorageOS volumes can be created on-demand through dynamic provisioning.
 
-1. Adminstrators create storage classes to define different types of storage,
-using a secret to authenticate against the StorageOS API.
-1. Users create a persistent volume claim and references it in a pod.
+1. Create the secret needed to authenticate against the StorageOS API.
+1. Adminstrators create storage classes to define different types of storage.
```

Comment:
The api secret and storage class are created by the Helm chart, so this only applies if using the container install method.  I'd move these to the container install method doc?

Reply:
I've added a note to skip to step 3 if you used the helm chart.